source/strings.tex

```diff
@@ -1345,25 +1327,17 @@
 
 \pnum
 \effects
-Constructs an object of class
-\tcode{basic_string}
+Constructs an object of class \tcode{basic_string}
 and determines its initial string value from the array of
-\tcode{charT}
-of length
-\tcode{traits::length(s)}
-whose first element is designated
-by \tcode{s},
-as indicated in Table~\ref{tab:strings.ctr.4}.
-
-\begin{libefftabvalue}
-{\tcode{basic_string(const charT*, const Allocator\&)} effects}
-{tab:strings.ctr.4}
-\tcode{data()}      &
-points at the first element of an allocated copy of the array whose first element
-is pointed at by \tcode{s}                                                              \\
-\tcode{size()}      &   \tcode{traits::length(s)}                                       \\
-\tcode{capacity()}  &   a value at least as large as \tcode{size()}                     \\
-\end{libefftabvalue}
+\tcode{charT} of length \tcode{traits::length(s)}
+whose first element is designated by \tcode{s}.
+
+\pnum
+\postconditions
+\tcode{data()} points at the first element of an allocated copy
+of the array whose first element is pointed at by \tcode{s},
+\tcode{size()} is equal to \tcode{traits::length(s)}, and
+\tcode{capacity()}  is a value at least as large as \tcode{size()}.
 \end{itemdescr}
```

Comment:
Duplicate space after capacity()? (Not that it matters.)

Reply:
Thanks, will fix.